### ISSUE TITLE
Windows name library glfw

### ISSUE THREAD

**Issue by bbvrpz10k** [jpbruyere/glfw-sharp#1]
Name library glfw in Windows OS "glfw3.dll"

**Comment by jpbruyere** [comment]
So I have implemented the new resolver of netcore3, but on linux, there's no way to debug netcore assemblies. So maybe I'll find another solution. 
```csharp
static Glfw3 () {
  NativeLibrary.SetDllImportResolver (Assembly.GetExecutingAssembly (),
    (libraryName, assembly, searchPath) => NativeLibrary.Load (
      Environment.OSVersion.Platform == PlatformID.Unix ? libraryName : libraryName + "3", assembly, searchPath));
```

**Comment by jpbruyere** [comment]
I'm testing (https://github.com/Firwood-Software/AdvancedDLSupport), this maybe a solution for universal dll mapping.

**Comment by jpbruyere** [comment]
resolved in 0.2.6-beta
